server/service/service_campaigns.go

```diff
@@ -85,6 +86,14 @@ func (svc Service) NewDistributedQueryCampaign(ctx context.Context, queryString
 		return nil, errors.Wrap(err, "new campaign")
 	}
 
+	defer func() {
+		var numHosts uint = 0
+		if campaign != nil {
+			numHosts = campaign.Metrics.TotalHosts
+		}
+		logging.WithExtras(ctx, "sql", queryString, "query_id", queryID, "numHosts", numHosts)
```

Comment:
```suggestion
		logging.WithExtras(ctx, "sql", queryString, "query_id", queryID, "num_hosts", numHosts)
```
(unless the existing log used this capitalization?)

Reply:
It's the way it was before.